CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.1.0
+* NEW: Added TypeScript declaration file for compatibility with TypeScript projects. See: https://github.com/getsentry/raven-js/pull/610
```

Comment:
Yeah, I'm torn. That, to me, falls under something like, us adding a feature into a plugin. Does that mandate a minor version bump for something that affects a small % of users? I dunno.